data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -345,63 +344,47 @@ def prepare_germline_jobs(this_batch: Batch, this_batch_run: BatchRun, this_sqr:
     job_list = []
     fastq_list_rows: List[dict] = libjson.loads(this_batch.context_data)
 
-    # get metadata for determining which sample need to be run through the germline workflow
-    metadata: dict = demux_metadata.handler({
-        'gdsVolume': this_sqr.gds_volume_name,
-        'gdsBasePath': this_sqr.gds_folder_path,
-        'gdsSamplesheet': this_sqr.sample_sheet_name,
-    }, None)
-
-    metadata_df = pd.DataFrame(metadata)
-    fastq_list_df = pd.DataFrame(fastq_list_rows)
-
-    # Add the sample_library_names attribute to fastq_list_df
-    fastq_list_df["sample_library_names"] = fastq_list_df['rgid'].apply(lambda x: x.rsplit('.', 1)[-1])
-
     # iterate through each sample group by rglb
-    for sample_library_name, sample_df in fastq_list_df.groupby("sample_library_names"):
+    for row in fastq_list_rows:
+        rgid = row['rgid']
 
-        # skip Undetermined samples
-        if sample_library_name.startswith("Undetermined"):
-            logger.warning(f"SKIP '{sample_library_name}' SAMPLE GERMLINE WORKFLOW LAUNCH.")
+        if row['read_1'].startswith("Undetermined"):
```

Comment:
I think, `read_1` start with `gds://` -- IIRC,
- It takes [read_1's location](https://github.com/umccr/data-portal-apis/blob/dev/data_processors/pipeline/lambdas/fastq_list_row.py#L100) 
- from bcl convert output, [example here](https://github.com/umccr-illumina/cwl-iap/blob/master/.github/tool-help/development/wfl.84abc203cabd4dc196a6cf9bb49d5f74/3.7.5.md#outputs-1) 

Having said, `Undetermined` are now stripped at pre BCL Convert workflow run? So this won't even need to check anymore?

Reply:
Yup, good point.
Can either change to a contains logic or completely remove it if Undetermined get stripped off anyway.

Reply:
@alexiswl can you confirm that the FastqListRow records don't contain records for the `Undetermined`?

Reply:
Confirmed, records won't contain the record 'Undetermined' because they're deleted prior to upload at the end of the TES task.

Reply:
Removed unnecessary statement in afb677c